test/draw/RasterRgbTest.ooc

```diff
@@ -84,27 +84,31 @@ RasterRgbTest: class extends Fixture {
 			image := RasterRgb open(this sourceSpace)
 			image2 := image copy()
 
-			image _coordinateSystem = CoordinateSystem YUpward
+			image referenceCount decrease()
+			image = RasterRgb open(this sourceSpace, CoordinateSystem YUpward)
```

Comment:
How does this test work, you open the same image over and over but you set a different coordinate system for it each time?